exec/java-exec/src/main/java/org/apache/drill/exec/physical/base/GroupScan.java

```diff
@@ -35,6 +35,8 @@
 public interface GroupScan extends Scan, HasAffinity{
 
   public static final List<SchemaPath> ALL_COLUMNS = ImmutableList.of(SchemaPath.getSimplePath("*"));
+  public static final List<SchemaPath> EMPTY_COLUMNS = ImmutableList.of();
```

Comment:
Nice catch. It's no longer needed. (Originally, I intended to convert NULL to empty_columns. But not it's not necessary). I'll remove that. thx.